applications/luci-app-opkg/htdocs/luci-static/resources/view/opkg.js

```diff
@@ -533,9 +533,9 @@ function renderDependencyItem(dep, info)
 		var text = pkg.name;
 
 		if (pkg.installsize)
-			text += ' (%.1024mB)'.format(pkg.installsize);
+			text += ' (%1024mB)'.format(pkg.installsize);
 		else if (pkg.size)
-			text += ' (~%.1024mB)'.format(pkg.size);
+			text += ' (~%1024mB)'.format(pkg.size);
```

Comment:
Should it not be `MiB`? Also would it not be better then to convert to Mb instead?